apps/records/views.py

```diff
@@ -125,6 +125,27 @@ def deleteRecord(request, pk):
 
 
 
+@login_required(login_url='login_page')
+@allowed_users(allowed_roles=['admins', 'monetary_users'])
+def createRecordModal(request):
```

Comment:
https://www.python.org/dev/peps/pep-0008/ -> 
```suggestion
def create_record_modal(request):
```